src/android/PowerManagement.java

```diff
@@ -115,12 +122,9 @@ public boolean execute(String action, JSONArray args,
 		try {
 			if( action.equals("acquire") ) {
 				if( args.length() > 0 && args.getBoolean(0) ) {
-					Log.d("PowerManagementPlugin", "Partial wake lock" );
-					result = this.acquire( PowerManager.PARTIAL_WAKE_LOCK );
-					handler.postDelayed(heartbeat, 10000);
-				}
-				else {
-					result = this.acquire( PowerManager.FULL_WAKE_LOCK );
+					result = partialWakeLock();
```

Comment:
yes, since when starting in an ionic project, we always get into the “else” construction where there was a complete wake lock.  Even if there are arguments or not.
I'm not good at Java, but when removing “if” and “else” in «if (action.equals ("acquire"))» it didn't compile because catch won't work, I decided to leave it that way, because everything works in my case